requirements.txt

```diff
@@ -9,4 +9,7 @@ python-dateutil
 time_machine
 shapely
 geopy
-pyproj
\ No newline at end of file
+pyproj
+numpy
+requests
+pytz
```

Comment:
Is pytz used? I don't see any references to it other than in here.

Reply:
When I tried to deploy the function without this it errored out, very confusing. I think it is required for the cloud build or something